### PR TITLE
my first commit ever

### PR DESCRIPTION
**Pull request by markstepchin**
first pull request ever

### CODE REVIEW

**Comment by thisiserik** [comment]
@markstepchin Awesome job, man! 
So the idea behind a pull request is for your team to discuss the work you did.

**Comment by thisiserik** [comment]
Usually I will look through your code and offer suggestions as to what you can improve. This is called code review. That's why we also work with branches. You put your work in a separate "environment" and through a pull request merge your work with everyone else's.

**Comment by markstepchin** [comment]
Makes sense

File: app/javascript/packs/app.jsx [hunk 1]
```diff
@@ -2,7 +2,11 @@ import React from 'react';
 import ReactDOM from 'react-dom';
```
Comment (thisiserik) [inline app/javascript/packs/app.jsx]: In the ES6 version of javascript, this is how we import packages to use into our file. It usually follows this convention: 
```js
import package from 'package-source`; 
```

File: app/javascript/packs/app.jsx [hunk 2]
```diff
@@ -2,7 +2,11 @@ import React from 'react';
...
 
 const App = () => (
```
Comment (thisiserik) [inline app/javascript/packs/app.jsx]: `cont App = () => (blah)` is one way of defining a function in ES6. We'll learn more about this later.

File: app/javascript/packs/app.jsx [hunk 3]
```diff
@@ -2,7 +2,11 @@ import React from 'react';
...
-  <div>Hello World!</div>
+  <div>
+    <h1>Erik is awesome!</h1>
+
+    <p>he really is</p>
+  </div>
```
Comment (thisiserik) [inline app/javascript/packs/app.jsx]: Great work! So this is actual HTML inside of your JS and it's called JSX. It's a cool way to mix JS and HTML code in one place.

Comment (markstepchin) [reply app/javascript/packs/app.jsx]: for sure man!

**Pull request merged**